node/src/main/scala/coop/rchain/node/api/DeployGrpcServiceV1.scala

```diff
@@ -291,5 +295,25 @@ object DeployGrpcServiceV1 {
             }
           )
           .flatMap(Observable.fromIterable)
+
+      def status(request: com.google.protobuf.empty.Empty): Task[StatusResponse] =
+        Monixable.apply.toTask(for {
+          address <- RPConfAsk[F].ask
+          apiErr  <- BlockAPI.getBlocks(1, 1)
+          blocks  = apiErr.right.getOrElse(List[LightBlockInfo]())
+          shardId = blocks.headOption.getOrElse(LightBlockInfo()).shardId
+          peers   <- ConnectionsCell[F].read
+          nodes   <- NodeDiscovery[F].peers
+          status = Status(
+            version = 1,
+            VersionInfo.get,
+            address.local.toAddress,
+            networkId,
+            shardId,
+            peers.length,
+            nodes.length
+          )
+          response = StatusResponse().withStatus(status)
+        } yield response)
```

Comment:
There is defined [extension method for `F[A]` with Monixable implicit](https://github.com/rchain/rchain/blob/a3807977676d3dbebc8cb30787ed2c0125a35fe5/shared/src/main/scala/coop/rchain/monix/Monixable.scala#L22) in scope and it's imported with `import coop.rchain.shared.syntax._`.

So this should work also:
```scala
(for { ... } yield response).toTask
```
---

Note also that definition of _apply_ functions means additional constructor so these two expressions have the same meaning.
```scala
// Directly use apply
Monixable.apply[F].toTask(...)
// or as a constructor like
Monixable[F].toTask(...)
```
In both cases Monixable instance must be present as implicit in scope.